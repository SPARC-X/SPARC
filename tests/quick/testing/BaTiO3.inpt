# Test: BaTiO3 #
CELL: 7.63 7.63 7.63
FD_GRID: 15 15 15
BC: P P P
EXCHANGE_CORRELATION: LDA_PZ
TOL_SCF: 1e-5
TOL_PSEUDOCHARGE: 1e-5
PRINT_FORCES: 1
PRINT_ATOMS: 1


