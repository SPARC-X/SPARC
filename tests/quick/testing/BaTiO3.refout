***************************************************************************
*                           SPARC (May 11, 2020)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun May 17 16:34:00 2020                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.63 7.63 7.63 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Gaussian
SMEARING: 0.007349864435
CHEB_DEGREE: 17
NSTATES: 29
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-05
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-05
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.59E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: BaTiO3
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 1
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 1
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  1
Mesh spacing                       :  0.508667 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  BaTiO3.out
Total number of atom types         :  3
Total number of atoms              :  5
Total number of electrons          :  40
Atom type 1  (valence electrons)   :  Ba 10
Pseudopotential                    :  ../Psp_for_tests/Ba.psp8
Pseudocharge radii of atom type 1  :  6.61 6.61 6.61 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  Ti 12
Pseudopotential                    :  ../Psp_for_tests/Ti.psp8
Pseudocharge radii of atom type 2  :  6.10 6.10 6.10 (x, y, z dir)
Number of atoms of type 2          :  1
Atom type 3  (valence electrons)   :  O 6
Pseudopotential                    :  ../Psp_for_tests/O.psp8
Pseudocharge radii of atom type 3  :  6.61 6.61 6.61 (x, y, z dir)
Number of atoms of type 3          :  3
Estimated total memory usage       :  5.20 MB
Estimated memory per processor     :  5.20 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.6459680804E+01        1.984E-01        0.136
2            -2.6386088591E+01        6.049E-02        0.037
3            -2.6380296280E+01        4.170E-02        0.037
4            -2.6379718071E+01        3.967E-02        0.036
5            -2.6377418199E+01        1.469E-02        0.036
6            -2.6377082173E+01        3.290E-03        0.037
7            -2.6377073217E+01        2.513E-03        0.038
8            -2.6377069113E+01        1.388E-03        0.038
9            -2.6377067009E+01        6.710E-04        0.038
10           -2.6377066593E+01        8.775E-05        0.035
11           -2.6377066827E+01        7.821E-05        0.037
12           -2.6377066371E+01        1.029E-05        0.036
13           -2.6377066756E+01        4.697E-06        0.031
Total number of SCF: 13    

k = [0.000, 0.000, 0.000]
Occupation of state 26 = 0.000000000000000.
Occupation of state 29 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.6377066642E+01 (Ha/atom)
Total free energy                  : -1.3188533321E+02 (Ha)
Band structure energy              : -1.1200045596E+01 (Ha)
Exchange correlation energy        : -2.1485252894E+01 (Ha)
Self and correction energy         : -2.0340676450E+02 (Ha)
Entropy*kb*T                       : -1.1232753484E-08 (Ha)
Fermi level                        :  2.8846432053E-01 (Ha)
RMS force                          :  4.7571725333E-01 (Ha/Bohr)
Maximum force                      :  8.5966383269E-01 (Ha/Bohr)
Time for force calculation         :  0.013 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.622 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
