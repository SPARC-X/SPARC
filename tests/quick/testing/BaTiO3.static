***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Ba:
      0.9882651394       0.9500437390       0.9854007162
Fractional coordinates of Ti:
      0.4883051330       0.5082999951       0.4491430331
Fractional coordinates of O:
      0.5311585887       0.9423479712       0.5054895152
      0.9293054969       0.4556304936       0.4885723786
      0.4720456043       0.4781589886       0.0261939929
