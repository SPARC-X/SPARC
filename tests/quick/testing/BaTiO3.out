***************************************************************************
*                       SPARC (version Oct 21, 2020)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Mar 18 15:55:39 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.63 7.63 7.63 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Gaussian
SMEARING: 0.007349864435
CHEB_DEGREE: 17
NSTATES: 29
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-05
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-05
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.59E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: BaTiO3
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 6
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 2 2
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  6
Mesh spacing                       :  0.508667 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  BaTiO3.out
Total number of atom types         :  3
Total number of atoms              :  5
Total number of electrons          :  40
Atom type 1  (valence electrons)   :  Ba 10
Pseudopotential                    :  Ba.psp8
Pseudocharge radii of atom type 1  :  6.61 6.61 6.61 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  Ti 12
Pseudopotential                    :  Ti.psp8
Pseudocharge radii of atom type 2  :  6.10 6.10 6.10 (x, y, z dir)
Number of atoms of type 2          :  1
Atom type 3  (valence electrons)   :  O 6
Pseudopotential                    :  O.psp8
Pseudocharge radii of atom type 3  :  6.61 6.61 6.61 (x, y, z dir)
Number of atoms of type 3          :  3
Estimated total memory usage       :  5.20 MB
Estimated memory per processor     :  887.70 kB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
